runtime/metrics/metrics.go

```diff
@@ -210,11 +223,15 @@ func (m *metricInfo[V]) getTS(labels any) (ts *timeseries[V], setup bool) {
 
 	// Initialize the values if they haven't yet been set up.
 	if !setup {
-		n := m.reg.numSvcs
 		if m.svcNum > 0 {
-			n = 1
+			ts.value = make([]V, 1)
+			ts.valid = make([]atomic.Bool, 1)
+			ts.valid[0].Store(true)
```

Comment:
Is this needed? Won't it happen in the other code above already?

Reply:
It would, yeah. My thinking was that it'd be useful to know if a metric defined within a service is zero (as opposed to not emitting that metric until it's increased/set).

Reply:
Ah, makes sense :) Thanks!